### DISCUSSION TITLE
Feature Request: [spatial] Hive Partitioning for Spatial formats

### DISCUSSION THREAD

**Discussion by Kanahiro** [duckdb/duckdb#10095] [Ideas]
Currently following SQL occurs segmentation fault 11

```sql
create table spatialdata as select * from 'spatialdata.fgb'; 
COPY spatialdata TO 'directory' (format gdal, driver 'FlatGeobuf', partition_by (column_name));
```

It seems that hive partitioning is not supported yet in spatial format.
It may be interesting to support to do so. what do you think?

**Comment by Maxxen** [comment]
Hi! I _think_ this is already fixed in the latest version of spatial (https://github.com/duckdb/duckdb_spatial/pull/202)

You can get the latest spatial version (built every time a PR is merged to main in the duckdb_spatial repo) for DuckDB 0.9.1 by executing:

```
FORCE INSTALL spatial FROM 'http://nightly-extensions.duckdb.org';
```

Note that this is a newer version than the one you get if you build DuckDB from source (eg 0.9.3-dev)

**Reply by Maxxen** [reply]
Right, the file extension will be .gdal and not fgb, requires a patch in duckdb but will look into it after the holidays!

**Reply by Kanahiro** [reply]
Thank you for kind response, it sounds great! Have a nice holiday 👍
